content/develop/connect/clients/client-side-caching.md

```diff
@@ -0,0 +1,187 @@
+---
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+description: Server-assisted, client-side caching in Redis
+linkTitle: Client-side caching
+title: Client-side caching introduction
+weight: 20
+---
+
+*Client-side caching* is a technique to reduce network traffic between
+a Redis client and the server. This generally gives better performance.
+
+By default, an [application server](https://en.wikipedia.org/wiki/Application_server)
+(which sits between the user app and the database) contacts the
+Redis database server through the client library for every read request.
+The diagram below shows the flow of communication from the user app,
+through the application server to the database and back again:
+
+{{< image filename="images/csc/CSCNoCache.drawio.svg" >}}
+
+When you use CSC, the client library
+maintains its own local cache of data items as it retrieves them
+from the database. When the same items are needed again, the client
+can satisfy the read requests from the cache instead of the database:
+
+{{< image filename="images/csc/CSCWithCache.drawio.svg" >}}
+
+Accessing the cache is much faster than communicating with the database over the
+network. Also, this technique reduces the load on the database server, so you may
+be able to run it using fewer nodes.
```

Comment:
Using fewer resources? In Enterprise and CE there are different approaches to address scalability (more shards, more nodes). Also, we can mention that CSC achieves lowering bandwidth consumption and expenses